src/adaptive_graph_of_thoughts/tests/test_exa_search_client.py

```
@@ -83,4 +83,57 @@
     with patch.object(ExaSearchClient, "_send_request") as mock_send:
         with pytest.raises(ValueError):
             client.query("bad params", num_results="ten")
-        mock_send.assert_not_called()
+        mock_send.assert_not_called()
+import json
```

Ruff (0.11.9)

Module level import not at top of file

(E402)